### DISCUSSION TITLE
Create Collections By Title?

### DISCUSSION THREAD

**Discussion by pressedhot** [meisnate12/Plex-Meta-Manager#222] [Q&A] [answered]
I am sorry if this is an easy question to answer but I've spent the past few hours trying many different combinations that would add Movies to a collection, based off a word or phrase found within the title and have had no luck.. 

So is it possible to achieve this or have I wasted my time even trying?

I am trying to have any files with "**How to Golf**" within the title, added to the "How to Golf" collection and here is an example of Movie titles that are within the library: 

- How to Golf - Part 1
- How to Golf - Part 2 
- How to Golf - Part 3 
- How to Golf - Part 4 

Any help or input would be greatly appreciated!

**Comment by meisnate12** [comment, marked as answer]
use a [Title Plex Search](https://github.com/meisnate12/Plex-Meta-Manager/wiki/Plex-Builders#plex-search)

```yaml
collections:
  How to Golf:
    plex_search:
      title: How to Golf
```
